apps/examples/grpc_greeter_client/Kconfig

```diff
@@ -5,6 +5,7 @@
 
 config EXAMPLES_GREETER_CLIENT
 	bool "grpc client example"
+    depends on GRPC
```

Comment:
Could you align new line? Tab instead of spaces makes it align

Reply:
Thanks for your comment. I will fix

Reply:
And it will be fixed by another pull request so I will reset it